fio.spec

```
@@ -60,6 +60,7 @@ EXTFLAGS="$RPM_OPT_FLAGS" make V=1 %{?_smp_mflags}
 
 %install
 rm -rf $RPM_BUILD_ROOT
+sed -i -e '1/s/python/python3/' %{buildroot}/usr/{src/fio-3.20/{t/{strided,steadystate_tests},.tools/hist/fio-histo-log-pctiles},bin/fio-histo-log-pctiles}.py
```

I think this should actually be:

```suggestion
sed -i -e '1s/python/python3/' %{buildroot}/usr/{src/fio-3.20/{t/{strided,steadystate_tests},.tools/hist/fio-histo-log-pctiles},bin/fio-histo-log-pctiles}.py
```
and it needs to be done after all of the installation steps (i.e. `make install`, etc.) so better done at the end of `%install` rather than the start of it.